src/main/java/org/snomed/quality/validator/mrcm/ValidationView.java

```diff
@@ -0,0 +1,33 @@
+package org.snomed.quality.validator.mrcm;
+
+import java.util.*;
+
+public enum ValidationView {
+
+	STATED("Stated"),
+	INFERRED("Inferred");
+
```

Comment:
Good idea to have enum for this. But I am not sure about the name. Maybe ContentType sounds better. What do you think?

Reply:
Fixed.